src/app/series/directives/series-podcast.component.html

```diff
@@ -27,7 +27,7 @@
       </div>
       <div class="span-fields">
         <prx-fancy-field label="Category" [model]="podcast" name="category"
-          [select]="categories" (change)="setSubCategories()" small=1>
+          [select]="categories" (onChange)="setSubCategories()" small=1>
```

Comment:
I think I see what you're doing here to hook this into the fancy field's prx-select rather than the select change event. Angular actually recommends these EventEmitter outputs not be named `onEvent` and instead just `event`. That doesn't seem to work here, but just mentioning it.

Reply:
Yeah, and for whenever reason, this immediately stopped working when the underlying field went from a real `<select>` to a fake dropdown.  So I'm guessing the `(change)` event was actually coming from the underlying select, not fired by the FancyFieldComponent itself, and that's why it used to work?

Reply:
(Maybe we need to stop event propagation in the fancy field component?)